src/Tests/Tests.Core/ManagedElasticsearch/Clusters/DirectStreamingReadOnlyCluster.cs

```diff
@@ -0,0 +1,15 @@
+using Elastic.Managed.Ephemeral.Plugins;
+using Nest;
+using Tests.Core.ManagedElasticsearch.NodeSeeders;
+
+namespace Tests.Core.ManagedElasticsearch.Clusters
+{
+	public class DirectStreamingReadOnlyCluster : ClientTestClusterBase
```

Comment:
Seems a *bit* heavy handed, can we just override in the test?

(Although I can see a counter argument here that we may want to keep around for other tests).